### PR TITLE
P/brentpicasso/track status and menu fixes

### PR DESCRIPTION
**Pull request by brentpicasso**
* added track detection status to center of toolbar
* streamlined + fixed normalized colors for toolbar background and slide out menu
* various code cleanup

### CODE REVIEW

File: autosportlabs/racecapture/menu/mainmenu.kv [hunk 1]
```diff
@@ -19,7 +19,7 @@
     orientation: 'vertical'
     canvas.before:
         Color:
-            rgba: 0.15, 0.03, 0.03, 1
+            rgba: [0.1, 0.1, 0.1, 1]
```
Comment (stieg) [inline autosportlabs/racecapture/menu/mainmenu.kv]: Changing the color?

Comment (brentpicasso) [inline autosportlabs/racecapture/menu/mainmenu.kv]: Yes, the dark burgundy / red didn't match any of the other colors in the app. normalizing things.

File: autosportlabs/racecapture/status/statuspump.py [hunk 2]
```diff
@@ -38,9 +38,11 @@ def status_worker(self):
...
+
+    def _update_all_listeners(self, status):
+        for listener in self._listeners:
+            listener(status)
         
     def _on_status_updated(self, status):
-        Logger.trace('StatusPump: status updated')
-        for listener in self._listeners:
-            Clock.schedule_once(lambda dt: listener(status))
+        Clock.schedule_once(lambda dt: self._update_all_listeners(status))
```
Comment (stieg) [inline autosportlabs/racecapture/status/statuspump.py]: Fix newline plz

Comment (brentpicasso) [inline autosportlabs/racecapture/status/statuspump.py]: ? not sure what you mean, sorry.

File: toolbarview.kv [hunk 3]
```diff
@@ -1,68 +1,84 @@
 <ToolbarView>:
     orientation: 'horizontal'
+    canvas.before:
+        Color:
+            rgba: [0.1, 0.1, 0.1, 1.0]
```
Comment (stieg) [inline toolbarview.kv]: Are we copy/pasting this a lot?  Is there a way to have a common value that all can use?

File: toolbarview.py [hunk 4]
```diff
@@ -106,15 +118,39 @@ def on_rc_rx_decay(self, dt):
...
-            self.teleStatus = kvFind(self, 'rcid', 'teleStatus')
+            self.teleStatus = self.ids.teleStatus
         try:        
             self.teleStatus.color = self.telemetry_color[status]
         except:
             Logger.error("ToolbarView: Invalid telemetry status: " + str(status))
 
+    def _update_track_status(self, status_data):
+        try:
+            track_status = status_data['status']['track']
```
Comment (stieg) [inline toolbarview.py]: Anyway this could lead to an index exception?

File: autosportlabs/racecapture/status/statuspump.py (hunk 2 above)
Comment (stieg) [inline autosportlabs/racecapture/status/statuspump.py]: Below is end of file with a tab but no empty line.  Makes GIT cranky.

File: toolbarview.kv (hunk 3 above)
Comment (brentpicasso) [inline toolbarview.kv]: yes! We have a ColorScheme class. thanks.

**Comment by brentpicasso** [comment]
extracted more colors, made progress bar show up behind text of status. simplified messages.

**Pull request merged**